locksmith/package.json

```diff
@@ -7,7 +7,7 @@
     "predev": "NODE_ENV=development sequelize db:migrate",
     "dev": "nodemon server.js",
     "pretest": "NODE_ENV=test sequelize db:migrate",
-    "test": "jest --forceExit",
+    "test": "jest test.js --forceExit",
```

Comment:
Hum. I think we should not "force" the script... but rather cleanup how tests are done in locksmith?
I worry that if someone later adds a `newTest.js` file, the new tests won't run...